### PR TITLE
Added help and logging information to the CLI documentation

### PR DESCRIPTION
**Pull request by andytael**
Added help and logging information to the CLI documentation

### CODE REVIEW

**Review by markxnelson — APPROVED** [review]
please have @rhondaday review also

File: docs-source/spring/content/development/cli/_index.md [hunk 1]
```diff
@@ -7,35 +7,50 @@ microservices applications as well as bindings with the resources that they use.
...
 
 1. Expose the Oracle Backend for Spring Boot and Microservices Admin server that the CLI calls using this command:
 
     ```cmd
-    kubectl port-forward services/obaas-admin -n obaas-admin 8080:8080
+    kubectl port-forward services/obaas-admin -n obaas-admin 8080
     ```
 
-2. Start the CLI in interactive mode by running `oractl` from your terminal window. For example:
+1. Start the CLI in interactive mode by running `oractl` from your terminal window. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:33]: Why is this a 1?

File: docs-source/spring/content/development/cli/_index.md [hunk 2]
```diff
@@ -7,35 +7,50 @@ microservices applications as well as bindings with the resources that they use.
...
 Admin Server Commands
-       connect: Connect to the Oracle Backend Administration Service.
+       connect: Connect to the OBaaS Spring Cloud admin console.
 
 Application/Namespace Commands
        create: Create an application/namespace.
-       delete: Delete a entire application/namespace or service.       
 
 Built-In Commands
        help: Display help about available commands
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:56]: To be consistent, add a period.

File: docs-source/spring/content/development/cli/_index.md [hunk 3]
```diff
@@ -113,26 +136,31 @@ The following is an example development workflow using the CLI:
...
    ```
 
-2. Use the `create` command to create an application namespace (Kubernetes *namespace*). The application namespace provides a mechanism for isolating groups of resources, especially the microservices. Names of resources need to be unique within a application namespace, but not across application namespaces.
+### create
 
-   ```cmd
-   oractl:>help create
-   NAME
-          create - Create an application/namespace.
+Use the `create` command to create an application namespace (Kubernetes *namespace*). The application namespace provides a mechanism for isolating groups of resources, especially the microservices. Names of resources need to be unique within a application namespace, but not across application namespaces.
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:141]: In the last sentence, change to "... within an application ... "

File: docs-source/spring/content/development/cli/_index.md [hunk 4]
```diff
@@ -181,41 +214,47 @@ The following is an example development workflow using the CLI:
...
-          --help or -h
-          help for bind
-          [Optional]
-   ```
+Ask for Help
+       Slack: https://oracledevs.slack.com/archives/C03ALDSV272
+       E-mail: obaas_ww@oracle.com
+```
 
    > ATTENTION: The `service-name` is mandatory and used as the name for the Schema/User to be created. If you want to use a different Schema/User from the `service-name`, you must also submit the`username`.
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:259]: Change both instances of "Schema/User" to "schema or user"

In the last sentence, add a space between "the 'username'."

File: docs-source/spring/content/development/cli/_index.md [hunk 5]
```diff
@@ -227,27 +266,27 @@ The following is an example development workflow using the CLI:
        Schema {myserv} was successfully created and Kubernetes Secret {myapp/myserv} was successfully created.
        ```
 
-   2. Use the `bind update` command to **update** a already created database schema or user for the service.
+   1. Use the `bind update` command to **update** a already created database schema or user for the service.
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:269]: Change to "Use ... an already created ... for the service. For example:"

File: docs-source/spring/content/development/cli/_index.md [hunk 6]
```diff
@@ -227,27 +266,27 @@ The following is an example development workflow using the CLI:
...
 
-5. Use the `deploy` command to create, build, and push an image for the microservice and create the necessary deployment, service,
-   and secret Kubernetes resources for the microservice.
+### deploy
 
-   ```cmd
-   oractl:>help deploy
-   NAME
-          deploy - Deploy a service.
+Use the `deploy` command to create, build, and push an image for the microservice and create the necessary deployment, service, and secret Kubernetes resources for the microservice.
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:279]: "Use ... for the microservice. For example:"

File: docs-source/spring/content/development/cli/_index.md [hunk 7]
```diff
@@ -458,7 +514,7 @@ The following is an example development workflow using the CLI:
        } ]
        ```
 
-   2. Use the `config list` command, without any parameters, to list the services that have at least one configuration inserted in the Spring Cloud Config server. For example:
+   1. Use the `config list` command, without any parameters, to list the services that have at least one configuration inserted in the Spring Cloud Config server. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:517]: Should this be 2?

File: docs-source/spring/content/development/cli/_index.md [hunk 8]
```diff
@@ -474,7 +530,7 @@ The following is an example development workflow using the CLI:
 
        ```
 
-   3. Use the `config list [parameters]` command to list the parameters using parameters as filters. For example:
+   1. Use the `config list [parameters]` command to list the parameters using parameters as filters. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:533]: Should this be 3?

File: docs-source/spring/content/development/cli/_index.md [hunk 9]
```diff
@@ -497,7 +553,7 @@ The following is an example development workflow using the CLI:
        } ]
        ```
 
-   4. Use the `config update` command to update a specific configuration using the set of parameters:
+   1. Use the `config update` command to update a specific configuration using the set of parameters:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:556]: Should this be 4?

"Use ... set of parameters. For example:"

File: docs-source/spring/content/development/cli/_index.md [hunk 10]
```diff
@@ -536,7 +592,7 @@ The following is an example development workflow using the CLI:
        } ]
        ```
 
-   5. Use the `config delete` command to delete the application configuration from the Spring Cloud Config server using one of the following two options:
+   1. Use the `config delete` command to delete the application configuration from the Spring Cloud Config server using one of the following two options:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:595]: Should this be 5?

File: docs-source/spring/content/development/cli/_index.md [hunk 11]
```diff
@@ -536,7 +592,7 @@ The following is an example development workflow using the CLI:
...
 
        * Delete all configurations from a specific service using the filters `--service-name`, `--service-profile` and `--service-label`. The
        CLI tracks how many configurations are present in the Spring Cloud Config server and confirms the completed deletion. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:598]: What is the second option?

File: docs-source/spring/content/development/cli/_index.md [hunk 12]
```diff
@@ -588,86 +646,95 @@ GraalVM Compile Commands
...
 
-SYNOPSIS
-       compile-logs --batch String --help 
+    If omitted, then the last batch is considered by default.
 
-OPTIONS
-       --batch String
-       File ID returned from the compile command. If not provided by default, then the last file compiled.
-       [Optional]
+1. Use the `compile-logs` command to retrieve the logs that show the compilation progress. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:711]: Should this be 2?

File: docs-source/spring/content/development/cli/_index.md [hunk 13]
```diff
@@ -680,68 +747,80 @@ OPTIONS
...
+    ------------------------------------------------------------------------------------------------------------------------
+    Produced artifacts:
+    /uploads/demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b.temp/demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b.exec (executable)
+    ========================================================================================================================
+    Finished generating 'demo-0.0.1-SNAPSHOT.jar_24428206-7d71-423f-8ef5-7d779977535b.exec' in 31m 30s.
+    Compiling Complete.
+    ```
 
-   3. Use the `compile-download` command to download the generated **.exec** file. For example:
+1. Use the `compile-download` command to download the generated **.exec** file. For example:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:767]: Should this be 3?

File: docs-source/spring/content/development/cli/_index.md [hunk 14]
```diff
@@ -680,68 +747,80 @@ OPTIONS
...
-   4. Use the `compile-purge` command to delete all of the artifacts generated on the GraalVM compiler service after downloading the **.exec** file:
+    File downloaded successfully to: 
+    /Users/cdebari/demo-0.0.1-SNAPSHOT.jar.exec
+    ```
 
-```cmd
-   oractl:>help compile-purge
-   NAME
-       compile-purge - Delete a launched job.
+1. Use the `compile-purge` command to delete all of the artifacts generated on the GraalVM compiler service after downloading the **.exec** file:
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:800]: Should this be 4?

File: docs-source/spring/content/development/cli/_index.md [hunk 15]
```diff
@@ -680,68 +747,80 @@ OPTIONS
...
+        [Optional]
+
+        Ask for Help
+            Slack: https://oracledevs.slack.com/archives/C03ALDSV272
+            E-mail: obaas_ww@oracle.com
+    ```
+
+## Logging
+
+The logfile for `oractl` on Mac/Unix machine is stored in the `$HOME/config/orctl` directory. The filename is `oractl-cli-history.log`
```
Comment (rhondaday) [inline docs-source/spring/content/development/cli/_index.md:826]: logfile -> log file
Mac/Unix -> Mac or Unix
filename -> file name

**Review by rhondaday — COMMENTED** [review]
Please incorporate a few comments.

File: docs-source/spring/content/development/cli/_index.md (hunk 2 above)
Comment (andytael) [reply docs-source/spring/content/development/cli/_index.md:56]: This is in the source code. I'll fix it there

File: docs-source/spring/content/development/cli/_index.md (hunk 11 above)
Comment (andytael) [reply docs-source/spring/content/development/cli/_index.md:598]: Changed to numbers

File: docs-source/spring/content/development/cli/_index.md (hunk 12 above)
Comment (andytael) [reply docs-source/spring/content/development/cli/_index.md:711]: It renders two 2 on the live website

File: docs-source/spring/content/development/cli/_index.md (hunk 13 above)
Comment (andytael) [reply docs-source/spring/content/development/cli/_index.md:767]: It renders to 3 on the live website

File: docs-source/spring/content/development/cli/_index.md (hunk 14 above)
Comment (andytael) [reply docs-source/spring/content/development/cli/_index.md:800]: It renders to 4 on the live website.

**Review by rhondaday — APPROVED** [review]
Looks good.

**Pull request merged**